docs/user_api/index.rst

```diff
@@ -1,45 +1,53 @@
-User API
+.. currentmodule:: geocat.viz
+
+   User API
 ========
 
 Routines
 --------
 
 GeoCAT-viz Plotting Routines
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+.. currentmodule:: geocat.viz.taylor
 .. autosummary::
    :toctree: ./generated/
 
-   geocat.viz.TaylorDiagram
+   geocat.viz.taylor.TaylorDiagram
```

Comment:
This is not wrong, but I figured I'd mention that if you explicitly list the current module, you don't actually need to link the full file path like

```python
.. currentmodule:: geocat.viz.taylor
.. autosummary::
   :toctree: ./generated/

   TaylorDiagram
```

Reply:
Yeah, I saw that. I don't think you need to change it, but I figured I'd mention that this probably doesn't need to be like this.